support-frontend/assets/pages/paper-subscription-landing/components/content/deliveryTab.tsx

```diff
@@ -57,53 +57,63 @@ export const accordionContainer = css`
-export const ContentDeliveryFaqBlock = ({
+export function ContentDeliveryFaqBlock({
 	setTabAction,
 }: {
-	setTabAction: (arg0: PaperFulfilmentOptions) => void;
-}) => (
-	<FlexContainer cssOverrides={flexContainerOverride}>
-		<div css={faqsContainer}>
-			<p css={paragraph}>
-				If you live in Greater London (within the M25), you can use the
-				Guardian’s home delivery service. If not, you can use our{' '}
-				<LinkTo tab={Collection} setTabAction={setTabAction}>
-					subscription cards
-				</LinkTo>
-				.
-			</p>
-			<p css={paragraph}>
-				Select your subscription below and checkout. You&apos;ll receive your
-				first newspaper as quickly as five days from subscribing.
-			</p>
-			<div css={accordionContainer}>
-				<Accordion>
-					<TabAccordionRow
-						trackingId={accordionTrackingId}
-						label="Delivery details"
-					>
-						<p>
-							Your newspaper will arrive before 8am from Monday to Saturday and
-							before 8.30am on Sunday.
-						</p>
-						<p>
-							We can’t deliver to individual flats, or apartments within blocks
-							because we need access to your post box to deliver your newspaper.
-						</p>
-						<p>
-							You can pause your subscription for up to 36 days a year. So if
-							you’re going away anywhere, you won’t have to pay for the
-							newspapers that you miss.
-						</p>
-					</TabAccordionRow>
-				</Accordion>
+	setTabAction: (paperFulfilmentOption: PaperFulfilmentOptions) => void;
+}): JSX.Element {
+	return (
+		<FlexContainer cssOverrides={flexContainerOverride}>
+			<div css={faqsContainer}>
+				<p css={paragraph}>
+					If you live in Greater London (within the M25), you can use the
+					Guardian’s home delivery service. If not, you can use our{' '}
+					<LinkTo tab={Collection} setTabAction={setTabAction}>
+						subscription cards
+					</LinkTo>
+					.
+				</p>
+				<p css={paragraph}>
+					Select your subscription below and checkout. You&apos;ll receive your
+					first newspaper as quickly as five days from subscribing.
+				</p>
+				<div css={accordionContainer}>
+					<Accordion>
+						<TabAccordionRow
+							trackingId={accordionTrackingId}
+							label="Delivery details"
+						>
+							<p>
+								Your newspaper will arrive before 8am from Monday to Saturday
+								and before 8.30am on Sunday.
+							</p>
+							<p>
+								We can’t deliver to individual flats, or apartments within
+								blocks because we need access to your post box to deliver your
+								newspaper.
+							</p>
+							<p>
+								You can pause your subscription for up to 36 days a year. So if
+								you’re going away anywhere, you won’t have to pay for the
+								newspapers that you miss.
+							</p>
+						</TabAccordionRow>
+						{/* The Accordian expects multiple children so this hack will appease the ts compiler */}
+						<AccordionRow
+							label={''}
+							cssOverrides={css`
+								display: none;
+							`}
+						/>
```

Comment:
Again we could maybe remove this `<AccordionRow />` by wrapping the first in a list

Reply:
Unfortunately doesn't work here since we'd created our own `TabAccordionRow` which causes a mismatch in the expected properties

Reply:
I just had a go locally and it seems to work out - what's the mismatch saying?

Reply:
After a reboot of my IDE this did behave! The mismatch referred to a difference in the type, props and key of the element array